src/content/about/index.md

```diff
@@ -21,52 +53,77 @@ We're an educational resource, designed to help new users become familiar with E
+- Empower community members to make increasingly significant contributions
+- Facilitate greater diversity of contributions: code, content, design, translation, moderation
+- Keep the codebase modern, clean, and well-documented
 
 ## Core principles {#core-principles}
 
+We have some core principles that help guide us to accomplish our mission.
+
 ### 1. ethereum.org is a portal to Ethereum 🌏 {#core-principles-1}
 
 We want our users to have their interest piqued and their questions answered. So our portal needs to combine information, "magic moments" and links to the brilliant community resources that exist out there. The purpose of our content is to be an “onboarding portal” and not a substitute for the extensive resources that already exist. We're keen to support and integrate with community built resources, giving them more visibility and making them more discoverable.
-
 [Ethereum's community](/community/) is at the heart of this: we need to not just serve the community, but work with them and incorporate their feedback. The website isn't just for the community we have now but for the community we hope to grow into. We must remember our community is global, containing people from many languages, regions, and cultures.
 
 ### 2. ethereum.org is always evolving 🛠 {#core-principles-2}
 
 Ethereum and the community are always evolving, so ethereum.org will too. That's why the site has a simple design system & modular structure. We make iterative changes as we learn more about how people use the site and what the community wants from it.
-
 We're open source, with a community of contributors, so you can propose changes or help us out too.
-
 [Learn about contributing](/contributing/)
 
 ### 3. ethereum.org is not a typical product website 🦄 {#core-principles-3}
 
 Ethereum is a big thing: it includes a community, a technology, a set of ideas and ideologies, and more.
 This means the website needs to handle many different user journeys, from “a developer who wants a specific tool” and “a newcomer who just bought some ETH and doesn’t know what a wallet is"
-
 "What is the best website for a blockchain platform?" remains an open question - we are pioneers. Building this requires experimentation.
 
+## Product roadmap {#roadmap}
+
+To make our work more accessible and to foster more community collaboration, the ethereum.org core team publishes an overview of our quarterly roadmap goals.
+
+[View our 2023 Q1 product roadmap](https://github.com/ethereum/ethereum-org-website/issues/9090)
+
+**How's that sound?** We always appreciate feedback on our roadmap - if there's something you think we should work on, please let us know! We welcome ideas and PRs from anyone in the community.
+
+**Want to get involved?** [Learn more about contributing](/contributing/), [hit us up on Twitter](https://twitter.com/ethdotorg), or join the community discussions in [our Discord server](https://discord.gg/CetY6Y4).
```

Comment:
We have https://ethereum.org/discord set up... although I suppose this would remove the external arrow.

```suggestion
**Want to get involved?** [Learn more about contributing](/contributing/), [hit us up on Twitter](https://twitter.com/ethdotorg), or join the community discussions in [our Discord server](/discord/).
```

@minimalsm @konopkja Are we keeping track of these Discord links? Do we have a preference on which ones we use?

Reply:
not to my knowledge or i dont / probbly the link with our domain looks better, but not that important